transport/src/main/java/io/netty/channel/nio/NioEventLoop.java

```diff
@@ -337,39 +340,33 @@ public void run() {
         // Register all channels to the new Selector.
         int nChannels = 0;
         for (;;) {
```

Comment:
this outer for loop seem pretty useless to me now? can you if it's still of use at least document what it does? I looked briefly and it seems like we exit with a break right away? I think it was only there to overcome the CME that was caught here